templates/index.html

```diff
@@ -304,6 +304,14 @@ <h3 id="modalLabel">Add Asset</h3>
                 <h5 class="d-flex">
                     <b class="justify-content-center align-self-center">Schedule Overview</b>
                     <div class="ml-auto">
+                        <a id="display-on-button" class="btn btn-long btn-outline-dark" href="#">
+                            <i class="fas fa-eye pr-2"></i>
+                            Display on
```

Comment:
Could you add success or failed alert message, please?

Reply:
Not sure how to do this. And as https://github.com/Screenly/screenly-ose/pull/1278#issuecomment-564647990 states a successful command does not guarantee the display really switched off. Better would be to call `display_status` afterwards and read the actual status. But we need async UI update for this, but don't know how.